libraries/GCS_MAVLink/GCS_Param.cpp

```diff
@@ -47,11 +47,10 @@ GCS_MAVLINK::queued_param_send()
     const uint32_t tnow = AP_HAL::millis();
     const uint32_t tstart = AP_HAL::micros();
 
-    // use at most 30% of bandwidth on parameters. The constant 26 is
-    // 1/(1000 * 1/8 * 0.001 * 0.3)
-    const uint32_t link_bw = _port->bw_in_kilobytes_per_second();
+    // use at most 30% of bandwidth on parameters
+    const uint32_t link_bw = _port->bw_in_bytes_per_second();
 
-    uint32_t bytes_allowed = link_bw * (tnow - _queued_parameter_send_time_ms) * 26;
+    uint32_t bytes_allowed = link_bw * (tnow - _queued_parameter_send_time_ms) / 333;
```

Comment:
this looks odd, was the old value wrong? This is changing the value by 333*26==8658, I was expecting it to change by 1024

Reply:
for 1/3 of bandwidth at 57600 baud, we have 5760 bytes/s, which means we want to use at most 1920 bytes/sec. Per millisecond we want to use 1.92 bytes/ms
with this change we get 5760/333, which is 17.29 bytes/ms, which doesn't seem right

Reply:
UUPPS, yes, I missed a 3!! It should be 3333 instead of 333 (that's when one is doing PRs by retyping code ... grrr). SORRY!!!

so, let's recalculate to double check:
for a delta of 1 sec, (tnow - _queued_parameter_send_time_ms) is 1000, divded by 3333 makes it 0.3 = 30%, times 5760 bytes / sec makes it 1728 bytes/sec.
now you can argue if you want 30% or 33% or whatever around it

yes, as commented in the intro post, 26 was wrong, way too high

so, let's also try to double check what we got using the "old" math
link_bw was calculated to 57600/(9*1024) = 6.25 so 6 in ints
so for a delta of 1 sec we would have 6 * 1000 * 26, which is obviously quite a large number

Reply:
to also repeat that, this is obviously a significant change in behavior, and thus may need good testing on several systems to better know the consequences